### PR TITLE
Add ttui-header and ttui-footer

### PR DESCRIPTION
**Pull request by mjamescompton**
#### Summary

Include ttui-header and ttui-footer

#### Screenshots

https://github.com/TheThingsIndustries/lorawan-stack-docs/assets/46600603/b78d2393-453c-45d3-839a-eebfe5393a62

#### Changes

- Replace header with ttui-header
- Update some page styling and the search input 
- Replace footer with ttui-footer

#### Notes for Reviewers

To run this locally you will need to run `yarn link` and be running  `yarn watch-linked` on this branch of the design repository https://github.com/TheThingsIndustries/design/pull/216

In this repo `cd docs` and `yarn link @tti/design`.

For some reason this does not work with the minified js, for now in the this commit
[util: TEMPORARY remove js minify for yarn link](https://github.com/TheThingsIndustries/lorawan-stack-docs/pull/1265/commits/f64db2dc68bd7691267df4be5d40adbc2123221c) I remove the js minify, we should put this back for deployment

You can then run the project as normal with `make`

@KrishnaIyer there is a screen size where there is to much in the menu, maybe we could organise this into dropdown for desktop?

#### Checklist

- [ ] Scope: The referenced issue is addressed, there are no unrelated changes.
- [ ] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

**Comment by nejraselimovic** [comment]
hey @mjamescompton, can you pls tag someone to review this?

File: doc/config/_default/config.toml [hunk 1]
```diff
@@ -43,6 +43,6 @@ pygmentsUseClasses = true
     weight = 2
 
   [[menu.buttons]]
-    name = "Get The Things Stack"
+    name = "Get Started"
```
Comment (KrishnaIyer) [inline doc/config/_default/config.toml:46]: This is confusing. We have a Getting Started section in the docs and it's also in the header menu.

**Review by KrishnaIyer — CHANGES_REQUESTED** [review]
> To run this locally you will need to run yarn link and be running yarn watch-linked on this branch of the design repository https://github.com/TheThingsIndustries/design/pull/216

So the new TTUI version cannot be pulled using `make js.deps`?

> @KrishnaIyer there is a screen size where there is to much in the menu, maybe we could organise this into dropdown for desktop?

Can you elaborate?

**Comment by mjamescompton** [comment]
> > To run this locally you will need to run yarn link and be running yarn watch-linked on this branch of the design repository [TheThingsIndustries/design#216](https://github.com/TheThingsIndustries/design/pull/216)
> 
> So the new TTUI version cannot be pulled using `make js.deps`?
> 
> > @KrishnaIyer there is a screen size where there is to much in the menu, maybe we could organise this into dropdown for desktop?
> 
> Can you elaborate?

This how you can use your local ttui with your local web project. It telling yarn that the location of the ttui package is not from the remote path but the node_modules contains an alias to local path.

https://classic.yarnpkg.com/lang/en/docs/cli/link/

Once we have released the design repo you can update the version number of ttui and `make js.deps` will get the latest

**Comment by KrishnaIyer** [comment]
Ok I see. Then let's wait for the TTUI release and then check this. We don't want to merge this accidentally before that and break CI.

**Pull request closed without merging**